### ISSUE TITLE
Common approach to XML namespace

### ISSUE THREAD

**Issue by kerstarno** [SAA-SDT/SharedSchema#2]
While EAC-CPF uses attributes from the XML namespace (xml:base, xml:id, xml:lang), EAD3 has taken the decision to move away from XML and to use its own version of these attributes (base, identifier respectively id, lang).

**Comment by kerstarno** [comment]
Similar to the related question on XLink (https://github.com/SAA-SDT/SharedSchema/issues/1) conversations so far have reiterated the argumentation followed during the EAD revision, that inclusion of namespaces introduce "onerous and needless complexity [...] when processing XML" (quote from Preface to EAD3 Tag Library).

However, the case might be slightly different for XML than it is for XLink.

One aspect to consider in the context, is the attribute xml:id (from EAC-CPF) and its equivalent id as well as the additional identifier (in EAD3). 

The latter is meant to hold a number, code, or string (e.g. URI) that uniquely identifies the term being used in a controlled vocabulary, taxonomy, ontology, or other knowledge organisation system and thereby replaces authfilenumber from EAD 2002. On `<unitid>`, identifier is a machine-processable unique identifier for the descriptive component in which the element appears. 

Should the subteam end up with the recommendation to reintroduce XML namespace in EAD3, renaming of identifier needs to be considered in order to avoid confusion.

**Comment by kerstarno** [comment]
Suggestion to keep XML namespace, i.e. re-introduce in EAD3.

- Should be fine for base and lang.
- Should be fine for id, too (see https://github.com/SAA-SDT/eac-cpf-schema/issues/52 in this context).
- Reconsider name for identifier attribute in EAD3 - e.g. is there an option to change back to authfilenumber?

**Comment by kerstarno** [comment]
I've linked this issue to the according conversation within the EAC-CPF subteam.

**Comment by SJagodzinski** [comment]
EAC-CPF team discussion:
Removing the namespace simplifies the schema and xml handling.
There is no specific EAC-CPF requirement in respect with linking.

Reusing a common vocabulary for these attributes seems useful. TS-EAS definition of attributes like @id and @base might not be that obvious.

Decision on namespace has to be made with the shared schema team and EAD team in order to get a consistent solution for both schemas.

**Comment by kerstarno** [comment]
Agreed with the aspect of discussing this together with EAD and EAC-CPF teams. @SJagodzinski : seeing that EAC-CPF team might be the one requiring a decision earliest, would you want to have this conversation at the May online meeting or the meeting in Austin or in a different form (email?) independent of any meeting?
